* The GROFF_G macro adds a `g' nameprefix as soon as it sees an existing
  troff.  On Gentoo Linux, and Gentoo Prefix we don't care, and don't
  want it to do this if the host system has it by coincidence.
  <grobian@gentoo.org>

--- configure.ac
+++ configure.ac
@@ -109,7 +109,8 @@
 # other random stuff
 GROFF_BROKEN_SPOOLER_FLAGS
 GROFF_PAGE
-GROFF_G
+g=
+AC_SUBST(g)
 GROFF_TMAC
 GROFF_TARGET_PATH_SEPARATOR
 GROFF_HTML_PROGRAMS
